### ISSUE TITLE
rc.11 error on Safari < 15.4 in SSG

### ISSUE THREAD

**Issue by ennioVisco** [nuxt/framework#7738]
### Environment

-

### Reproduction

-

### Describe the bug

Running on the client on old safari versions requires `Crypto.RandomUUID` which has been introduced in Safari 15.4 (see: https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)
<img width="992" alt="image" src="https://user-images.githubusercontent.com/740025/191624801-55e4f5d3-c430-4098-8f60-ce6cea8d71d1.png">

**Comment by danielroe** [comment]
I don't believe this is coming from Nuxt. Would you provide a reproduction please? :pray:
